src/libs/PolicyUtils.ts

```diff
@@ -227,14 +227,14 @@ function isPaidGroupPolicy(policy: OnyxEntry<Policy> | EmptyObject): boolean {
  * Checks if policy's scheduled submit / auto reporting frequency is "instant".
  * Note: Free policies have "instant" submit always enabled.
  */
-function isInstantSubmitEnabled(policy: OnyxEntry<Policy>): boolean {
+function isInstantSubmitEnabled(policy: OnyxEntry<Policy> | EmptyObject): boolean {
```

Comment:
Is this to support the case when the policy is not loaded or what? 
And if so, why was it not needed before?

Reply:
It's same as other cases if you see next line.

`policy?.autoReportingFrequency`
policy has optional chaining for nothing.

This came from the fact that Onyx values are optional (null on fist composer render, before Onyx data is loaded)